docs/reference/filters.md

```diff
@@ -1695,6 +1695,128 @@ oidcClaimsQuery("/:name%\"*One\"", "/path:groups.#[%\"*-Test-Users\"] groups.#[=
 
 As of now there is no negative/deny rule possible. The first matching path is evaluated against the defined query/queries and if positive, permitted.
 
+### Open Policy Agent
+
+To get started with Open Policy Agent, also have a look at the [tutorial](../tutorials/auth.md#open-policy-agent). This section is only a reference for the implemented filters. 
+
+#### authorizeWithRegoPolicy
```

Comment:
I was not sure what REgo is, it turns out to be a policy language. Maybe we can add a link to https://www.openpolicyagent.org/docs/latest/policy-language/ here or in the tutorial.

I am also wondering about filter names: should we reflect OPA and omit Rego (as it is implied) in the name, maybe `opaAuthorizeRequest` and `opaServeResponse`?

Reply:
If we would have an inline policy filter we can also name it `opaAuthorizeInlineRequest` for example.
In case of a new WASM target we could introduce the target in the naming scheme and either alias opaAuthorizeRequest to opaRegoAuthorizeRequest (or something else). Maybe we could even detect the target from the input, so I think I am in favor of the suggested name by @AlexanderYastrebov .

Reply:
Renamed both filters.